packages/react-api/src/util/keyringFormat.ts

```diff
@@ -0,0 +1,15 @@
+import { KeyringPair$Json } from '@polkadot/keyring/types';
+import { KeyringPair$Json2 } from "@polkadot/react-api/types";
+
+export function updateToV3KeyringFormat(json: KeyringPair$Json | KeyringPair$Json2) {
```

Comment:
This function even though tiny does 3 jobs in one: Detection, Conversion and Update. The separation of these jobs will make it possibly a better idea. Especially that it's not hard.

Reply:
It's easier to maintain like this + don't plan on using the pieces independently at this point